lib/src/internal/http_endpoints.dart

```diff
@@ -337,7 +338,7 @@ abstract class IHttpEndpoints {
   Future<IDMChannel> createDMChannel(Snowflake userId);
 
   /// Used to send a request including standard bot authentication.
-  Future<IHttpResponse> sendRawRequest(String url, String method,
+  Future<HttpResponse> sendRawRequest(HttpRoute route, String method,
```

Comment:
Can be HttpRoute be hidden after interface with static method for instance creation? In nyxx we dont expose any internal classes and logic